Modules/RPC.cs

```diff
@@ -74,6 +74,7 @@ enum CustomRPC : byte // 193/255 USED
     SetLoversPlayers,
     SetExecutionerTarget,
     RemoveExecutionerTarget,
+    BetterCheck, // BetterAmongUs (BAU) RPC, This is sent to allow other BAU users know who's using BAU!
```

Comment:
Set this to a fixed value in case we add other custom rpcs in the future and change the callid